ansible_collections/arista/avd/molecule/eos_cli_config_gen/inventory/host_vars/monitor-telemetry-influx.yml

```
@@ -0,0 +1,29 @@
+monitor_telemetry_influx:
+  vrf: test
+  destinations:
+    - name: test
```

add an example for default
```suggestion
    - name: test_default
```